src/uniprotkb/components/statistics/AminoAcidBarPlot.tsx

```diff
@@ -5,9 +5,9 @@ import { sum } from 'lodash-es';
 import { StatisticsCategory } from './StatisticsPage';
 
 // Specify the chart’s dimensions.
-const width = 500;
+const width = 340;
 const height = 400;
-const margin = { top: 10, right: 30, bottom: 40, left: 40 };
+const margin = { top: 20, right: 30, bottom: 60, left: 40 };
```

Comment:
Looking at l. 104, the total width will end up being this width plus the 2 inline margins (410), so either the grid definition needs to be adjusted to create a new row at 410px instead of 400, or here the sum needs to stay at 400px maximum